src/lib/Components/Bidding/Screens/BidResult.tsx

```diff
@@ -49,7 +49,7 @@ export class BidResult extends React.Component<BidResultProps> {
     }
 
     // fetch the latest increments for the select max bid screen
-    if (this.props.refreshSaleArtwork()) {
+    if (this.props.refreshSaleArtwork) {
```

Comment:
The TS upgrade led to identifying this little bug fix, although I didn’t find any errors in Sentry, so presumably this was never undefined. However, the callback would get invoked twice, which may or may not have been an issue.